server/base/src/main/java/org/apache/accumulo/server/util/ZooZap.java

```diff
@@ -80,8 +89,21 @@ static class Opts extends Help {
     boolean zapCompactors = false;
     @Parameter(names = "-sservers", description = "remove scan server locks")
     boolean zapScanServers = false;
+    @Parameter(names = "--gc", description = "remove gc server locks")
+    boolean zapGc = false;
+    @Parameter(names = "--monitor", description = "remove monitor server locks")
+    boolean zapMonitor = false;
```

Comment:
`--gc` should probably be added to [MiniAccumuloClusterImpl](https://github.com/apache/accumulo/blob/2.1/minicluster/src/main/java/org/apache/accumulo/miniclusterImpl/MiniAccumuloClusterImpl.java#L840). Both should be added to [UpgradeUtil](https://github.com/apache/accumulo/blob/2.1/server/base/src/main/java/org/apache/accumulo/server/util/UpgradeUtil.java#L135)

Reply:
Maybe ZooZap should have a `--all` option.

Reply:
updated in d114731